worker/pkg/zip/zip.go

```diff
@@ -0,0 +1,48 @@
+package zip
+
+import (
+	"archive/zip"
+	"io"
+)
+
+type Unzipper struct {
+	r   *zip.Reader
+	wFn func(name string) (io.Writer, error)
+}
+
+func NewUnzipper(ra io.ReaderAt, size int64, wFn func(name string) (io.Writer, error)) (*Unzipper, error) {
```

Comment:
```suggestion
func NewUnzipper(r *zip.Reader, wFn func(name string) (io.WriteCloser, error)) (*Unzipper, error) {
```

- If a new parameter is added to `zip.NewReader`, `NewUnzipper` will also be affected. In these cases, there is a mixture of multiple interests.
- If it does not have the ability to close a Writer, users will have to save all the Writers they create in some other variable to ensure the Writer is closed.

Reply:
OK! 👍